app/views/custody_suite/defence_requests/form_partials/_detainee_details.html.erb

```diff
@@ -14,6 +14,7 @@
 <% end %>
 
 <%= f.text_field_input(:detainee_address,
+    multiline: 5,
```

Comment:
Related to the above comment having a text_area_input helper would be cleaner and better revealing about what the output from the helper method is going to be